config/components/prometheus-monitoring/kustomization.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

[Kustomize Components](https://kubectl.docs.kubernetes.io/guides/config_management/components/) should use a different version / kind so they can be included as a component. 

```suggestion
apiVersion: kustomize.config.k8s.io/v1alpha1
kind: Component
```